srcpkgs/spotify/template

```diff
@@ -14,7 +14,7 @@ maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
 license="custom:Proprietary"
 homepage="https://www.spotify.com"
 distfiles="http://repository.spotify.com/pool/non-free/s/spotify-client/${_filename}"
```

Comment:
Is the current version not downloadable because they remove old versions from their server? If yes, is there an archive that doesn't do that?

Reply:
Yes, they removed it from our servers.
About your question. Do you mean an archive like spotify-current.deb? In this case, no. 

 If you access the URL: http://repository.spotify.com/pool/non-free/s/spotify-client/

You will see these packages:
```
[spotify-client_1.0.72.117.g6bd7cc73-35_i386.deb](http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.0.72.117.g6bd7cc73-35_i386.deb)    27-Feb-2018 13:30            95135550
[spotify-client_1.1.26.501.gbe11e53b-15_amd64.deb](http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.1.26.501.gbe11e53b-15_amd64.deb)   19-Feb-2020 12:36           120102446
[spotify-client_1.1.72.439.gc253025e_amd64.deb](http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.1.72.439.gc253025e_amd64.deb)      12-Nov-2021 11:29           117803268
[spotify-client_1.1.84.716.gc5f8b819_amd64.deb](http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.1.84.716.gc5f8b819_amd64.deb)      27-Apr-2022 12:00           119770140
```

The last version was dropped.